scintilla/gtk/ScintillaGTK.cxx

```diff
@@ -179,6 +180,18 @@ GdkAtom SelectionOfGSD(GtkSelectionData *sd) noexcept {
 	return gtk_selection_data_get_selection(sd);
 }
 
+bool SettingGet(GtkSettings *settings, const gchar *name, gpointer value) noexcept {
```

Comment:
This looks like the only change that is likely to cause segfaults on widget destroy of the main window if these settings are added to the Scintilla widget and the memory is double released?